app/experimenter/experiments/migrations/0073_experiment_new_analysis_owner.py

```diff
@@ -0,0 +1,62 @@
+# Generated by Django 2.1.11 on 2019-10-03 21:53
+
+from difflib import SequenceMatcher as SM
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def closest_user(users, target_name):
+    return sorted(
+        [(SM(None, user.email, target_name).ratio(), user) for user in users],
+        reverse=True,
+    )[0][1]
+
+
+def forward_analysis_owner(apps, schema_editor):
+    Experiment = apps.get_model("experiments", "Experiment")
+    User = apps.get_model(*settings.AUTH_USER_MODEL.split("."))
+
+    for experiment in Experiment.objects.all():
+        experiment.new_analysis_owner = closest_user(
+            User.objects.all(), experiment.analysis_owner
+        )
```

Comment:
Sequencematcher is really cool!
Was playing around with it, it's case sensitive, would we want to lower analysis owner for better ratios?
```
>>> SM(None, "ttran@mozilla.com", "tif tran").ratio()
0.4
>>> SM(None, "ttran@mozilla.com", "Tif Tran").ratio()
0.24

```

Reply:
Oh good idea 👍